3.x: Driver does not build on fedora

Tested fedora 41, fedora 38.

Running `mvn verify` or `mvn verify -U` currently ends with error on those systems:
```
[ERROR] Failed to execute goal on project scylla-driver-core: Could not resolve dependencies for project com.scylladb:scylla-driver-core:jar:3.11.5.6-SNAPSHOT: Could not find artifact io.netty:netty-tcnative-boringssl-static:jar:linux-x86_64-fedora:2.0.70.Final in central (https://repo.maven.apache.org/maven2) 
```

--- a/README.md
+++ b/README.md
@@ -90,6 +90,10 @@ The 'extras' module is also published as a separate artifact:
 We also provide a [shaded JAR](manual/shaded_jar/)
 to avoid the explicit dependency to Netty.
 
+On Fedora 30 or later you may need to install dependencies for `netty-tcnative-boringssl-static`
+by running `dnf -y install libxcrypt-compat`
+
+
 ## Compatibility
 
 The Java client driver 3.11.5.0 ([branch scylla-3.x](https://github.com/scylladb/java-driver/tree/scylla-3.x)) is compatible with 

--- a/pom.xml
+++ b/pom.xml
@@ -97,7 +97,11 @@
         <javadoc.opts />
         <!-- Append "-fedora" to os.detected.classifier for Fedora systems
              (see https://netty.io/wiki/forked-tomcat-native.html) -->
+        <!-- Commenting this out since artifact in use right now is statically
+             linked. There is no linux-x86_64-fedora variant like with
+             regular netty-tcnative.
         <os.detection.classifierWithLikes>fedora</os.detection.classifierWithLikes>
+        -->
     </properties>
 
     <dependencyManagement>

